### ISSUE TITLE
Artificial reflections when using GPU mode

### ISSUE THREAD

**Issue by pasansherath** [geodynamics/specfem2d#1075]
Hi,

**Comment by danielpeter** [comment]
good point, so far the GPU implementation didn't consider mixed isotropic/anisotropic cases yet. it has been amended now, with PR #1076 

please update to the current devel version of the code and it should work - otherwise, let me know...

**Comment by pasansherath** [comment]
@danielpeter It works perfectly now. Thanks very much for solving this matter quickly. I am impressed with the performance using GPU mode. I was able to cut down the simulation time to about 5 hours using GPU, which took 200+ hours on 40 CPU cores.
